test/rules/no-dupe-feature-names/SecondFeature.feature

```diff
@@ -0,0 +1,13 @@
+Feature: second feature
```

Comment:
I think this file was added by accident (it's in the no-dupe-feature-names rule, which you didn't change). Do you mind deleting it?